src/software/world/ball_model/linear_ball_model.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <optional>
+
+#include "software/world/ball_model/ball_model.h"
+
+/**
+ * LinearBallModel is a BallModel that makes the following assumptions about the ball:
+ * 1. no initial spin
+ * 2. slides with a constant sliding friction above a certain speed threshold
+ * 3. rolls with a constant rolling friction below a certain speed threshold
+ * 4. the transition between sliding and rolling is instantaneous
+ */
+class LinearBallModel final : public BallModel
```

Comment:
Sorry, realized I forgot to comment on this last time. Could we rename this to something like `TwoStageBallModel`, or maybe `TwoStageLinearBallModel`? I would expect a `LinearBallModel` to just have constant decel?

Reply:
done